.github/workflows/pull-request.yml

```
@@ -44,10 +44,28 @@ jobs:
       
       - name: Build
         run: dotnet build Letterbook.sln --configuration Release
+
+      - name: Install Playwright Browsers
+        run: pwsh Letterbook.Web.Tests/bin/Release/net8.0/playwright.ps1 install
+
+      - name: Docker compose up
+        run: docker compose up -d
+
+      - name: Seed database
+        run: dotnet tool restore && dotnet ef database update --project Letterbook.Adapter.Db/Letterbook.Adapter.Db.csproj && dotnet ef database update --project Letterbook.Adapter.TimescaleFeeds/Letterbook.Adapter.TimescaleFeeds.csproj
```

```suggestion
        run: dotnet ef database update --project Letterbook.Adapter.Db/Letterbook.Adapter.Db.csproj && dotnet ef database update --project Letterbook.Adapter.TimescaleFeeds/Letterbook.Adapter.TimescaleFeeds.csproj
```

Tools should already be restored from a few steps ago